staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -67,7 +67,11 @@ func (d authenticatedDataString) AuthenticatedData() []byte {
 	return []byte(string(d))
 }
 
-var _ value.Context = authenticatedDataString("")
+var (
+	_ value.Context = authenticatedDataString("")
+	// RequestWatchProgress is an etcd feature that may use to check if it supported or not.
+	RequestWatchProgress storage.Feature = "RequestWatchProgress"
```

Comment:
Why this is here and not in storage package?

Reply:
We thought because it's an store feature we put it there, i'l move it to storage package